index.md

```diff
@@ -58,13 +59,13 @@ Interested? Check out:
 
 -   the [specification](documentation.md)
 -   some [examples](examples.md)
--   this [excellent guide](http://spacetelescope.github.io/understanding-json-schema/) for schema authors, from the [Space Telescope Science Institute](http://www.stsci.edu/)
 -   the growing list of [JSON (Hyper-)Schema software](implementations.md)
 
+We encourage updating to the latest specification, as described by the draft-06 meta-schemas.  However, if you are still on draft-04, you be be interested in:
```

Comment:
s/be be/may be/

and maybe:
"using draft-04" or "targeting draft-04" instead of "on draft-04"

Reply:
Good catch & good idea, thanks.